addons/project/models/project_project.py

```diff
@@ -446,6 +446,8 @@ def copy(self, default=None):
             default = {}
         if not default.get('name'):
             default['name'] = _("%s (copy)", self.name)
+        if not default.get('active') and not self.active:
```

Comment:
```suggestion
        if not default.get('active'):
```
Is this second condition necessary?